src/Filter/ModelFilter.php

```diff
@@ -55,7 +55,7 @@ public function getDefaultOptions()
             'field_name' => false,
             'field_type' => DocumentType::class,
             'field_options' => [],
-            'operator_type' => EqualType::class,
+            'operator_type' => EqualOperatorType::class,
```

Comment:
This type change is intentional?

Reply:
In this case and the one below it is, [EqualType is deprecated](https://github.com/sonata-project/form-extensions/blob/1.x/src/Type/EqualType.php#L20).

Reply:
You can compare with https://github.com/sonata-project/SonataDoctrineORMAdminBundle/blob/3.x/src/Filter/ModelFilter.php#L49 @jordisala1991